src/main/scala/com/bwsw/cloudstack/vault/server/kafka/Consumer.scala

```diff
@@ -0,0 +1,75 @@
+package com.bwsw.cloudstack.vault.server.kafka
+
+import java.util.concurrent.CountDownLatch
+import java.util.{Collections, Properties}
+
+import com.bwsw.cloudstack.vault.server.common.traits.EventHandler
+import org.apache.kafka.clients.consumer.{ConsumerConfig, KafkaConsumer}
+import org.slf4j.LoggerFactory
+
+import scala.collection.JavaConverters._
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success}
+
+/**
+  * Created by medvedev_vv on 31.07.17.
+  */
+class Consumer[T](val brokers: String,
+                  val topic: String,
+                  val groupId: String,
+                  val pollTimeout: Int,
+                  val eventHandler: EventHandler[T])
+                 (implicit executionContext: ExecutionContext) {
+
+  private val logger = LoggerFactory.getLogger(this.getClass)
+  private val props: Properties = createConsumerConfig(brokers)
+  private val consumer = new KafkaConsumer[String, String](props)
+
+  def createConsumerConfig(brokers: String): Properties = {
+    val props = new Properties()
+    props.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, brokers)
+    props.put(ConsumerConfig.GROUP_ID_CONFIG, groupId)
+    props.put(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, "true")
+    props.put(ConsumerConfig.AUTO_COMMIT_INTERVAL_MS_CONFIG, "10000")
+    props.put(ConsumerConfig.SESSION_TIMEOUT_MS_CONFIG, "30000")
+    props.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, "org.apache.kafka.common.serialization.StringDeserializer")
+    props.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, "org.apache.kafka.common.serialization.StringDeserializer")
+    props
+  }
+
+  def shutdown(): Unit = {
+    consumer.close()
+  }
+
+  def start() {
```

Comment:
It would be better to use "process" name since the starting in the infinite loop looks weird